core/src/main/java/io/spine/core/Events.java

```diff
@@ -257,6 +258,16 @@ public static RejectionEventContext rejectionContext(CommandMessage commandMessa
                                     .build();
     }
 
+    /**
+     * Obtains the type URL of the event message enclosed by a given {@code Event}.
+     */
+    public static TypeUrl typeUrl(Event event) {
```

Comment:
This is weird. I think it belongs to a message interface, such as `EventMessage`.

Also I suspect this isn't the first time we want to find out the `TypeUrl` of an `Event`. So please have a look around, maybe this task is already solved somehow.